tests/tests_integration/test_api/test_postgres_gateway/test_tables.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from jaraco.context import suppress
```

Comment:
Hmm, auto-imported by your IDE?
```suggestion
from contextlib import suppress
```